src/components/ShoppingList/index.tsx

```diff
@@ -1,18 +1,37 @@
-import { Button, Flex } from '@chakra-ui/react';
+import { Button, Flex, Badge } from '@chakra-ui/react';
 import { MdOutlineShoppingCart } from 'react-icons/md';
-
+import { useState } from 'react';
 export const ShoppingList = () => {
+  //A contagem será recuperada como prop de outro componente, o número aqui é meramente ilustrativo
+  const [count] = useState(4);
   return (
-    <Flex h="42px" w="42">
+    <Flex h="50px">
       <Button
         h="full"
         w="full"
         backgroundColor="#F9A109"
         borderRadius="50%"
         color={'white'}
       >
-        <MdOutlineShoppingCart fontSize={20} />
+        <MdOutlineShoppingCart fontSize={25} />
       </Button>
+      <Badge
+        textAlign="center"
+        ml="35px"
+        mt="-5px"
+        variant="solid"
+        color="white"
+        bg="red"
+        fontSize="16"
+        h="fit-content"
+        w="fit-content"
+        zIndex="1"
+        borderRadius="20%"
```

Comment:
Border radius of [figma](https://www.figma.com/file/kX8FNpYFVLI4ivuEQ6yGgS/Shopping-List?node-id=6-2&t=YN4OJ6h6nmbiKz05-0) is **4px**

```suggestion
        borderRadius="4px"
```